common-pages/src/TimeFilter/QuickSelects.tsx

```diff
@@ -389,8 +569,8 @@ export const AvailableQuickSelects: IQuickSelect[] = [
                 duration: 12
             }
         },
-        hideQuickPick: (f) => {
-            return f == 'time'
+        hideQuickPick: (f, s) => {
+            return f == 'time' || s == 'dateTimeShort'
```

Comment:
`f` represents the type of input (e.g. `Date` or `Time` or `datetime`) we should be using that instead of adding the `s`